src/OVAL/probes/probe-api.c

```diff
@@ -1405,7 +1405,7 @@ SEXP_t *probe_item_create(oval_subtype_t item_subtype, probe_elmatr_t *item_attr
         bool    multiplied;
         int     value_i, multiply;
 
-        subtype_name = oval_subtype_to_str(item_subtype);
+        subtype_name = oval_subtype_get_text(item_subtype);
```

Comment:
If `item_subtype` is really invalid, `oval_subtype_get_text()` will return `"**INVALID**"` and condition bellow will not recognize invalid subtype and will continue

```
const char *oval_subtype_get_text(oval_subtype_t subtype)
{
...
	if (map) {
		return oval_enumeration_get_text(map, subtype);
	} else {
		oscap_seterr(OSCAP_EFAMILY_OVAL, "Invalid OVAL family.");
		return _invalid;
	}
```

Please, tell me why it cannot happen or why the behavior wasn't changed with your changes.
If are there any hidden assumptions, maybe I would like to have some `assert()` in the code if it is possible.

Similar "issue" https://github.com/OpenSCAP/openscap/pull/458/files#diff-4fcc494830030cae0c83ae842a047a49R675